libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -495,6 +495,9 @@ public void track(AnalyticsTracker.Stat stat, Map<String, ?> properties) {
             case SITE_SETTINGS_DELETE_SITE_RESPONSE_ERROR:
                 eventName = "site_settings_delete_site_response_error";
                 break;
+            case ABTEST_START:
+                eventName = "abtest_start";
```

Comment:
Oh if they're using start for their event verb, might as well stick with it. Jerkfaces!